OrbitGl/TimeGraph.cpp

```diff
@@ -242,18 +242,20 @@ void TimeGraph::HorizontallyMoveIntoView(const TextBox* text_box,
 
 void TimeGraph::VerticallyMoveIntoView(const TextBox* text_box) {
   CHECK(text_box != nullptr);
-  // TODO: Sometimes the Y-coordinate is not set. We only need to update the
-  // Y-coordinate from this text box
-  UpdatePrimitives();
+  const Timer& timer = text_box->GetTimer();
+  auto thread_track = GetOrCreateThreadTrack(timer.m_TID);
+  // TODO: Could we need to update the depth of the timer?
```

Comment:
I don't think so, but not 100% sure.

Reply:
We shouldn't change the m_Depth member of Timer once it's set, no.

Reply:
The depth represents an index into a stack of instrumented functions, it should always remain the same.

Reply:
Yes, I know. And my doubt was that if it set/updated in UpdatePrimitives, maybe has never set at this moment.

Reply:
It is actually set already on each timer, so we should be fine. 

I vaguely recalled there was an UpdateDepth() method called in UpdatePrimitives, but that does not update the depth of the timer, but the (current) depth of the track .